market_maker_keeper/imtoken_pricing_server.py

```diff
@@ -95,40 +69,73 @@ def __init__(self, args: list, **kwargs):
+        with open(self.arguments.config) as json_file:
+            data = json.load(json_file)
 
-        pair = ImtokenPair(self.arguments.base_pair, self.arguments.counter_pair)
+        pairs, configs = self.parse_configs(data=data)
 
         application = tornado.web.Application([
-            (r"/pairs", PairsHandler, dict(pair=pair)),
-            (r"/indicativePrice", IndicativePriceHandler, dict(pair=pair,
-                                                               base_bands_config=self.base_bands_config,
-                                                               counter_bands_config=self.counter_bands_config,
-                                                               price_feed=self.price_feed,
-                                                               spread_feed=self.spread_feed,
-                                                               control_feed=self.control_feed,
-                                                               history=self.history,
+            (r"/pairs", PairsHandler, dict(token_pairs=pairs)),
+            (r"/indicativePrice", IndicativePriceHandler, dict(pairs=pairs,
+                                                               configs=configs,
                                                                cache=self.cache)),
-            (r"/price", PriceHandler, dict(pair=pair,
-                                           base_bands_config=self.base_bands_config,
-                                           counter_bands_config=self.counter_bands_config,
-                                           price_feed=self.price_feed,
-                                           spread_feed=self.spread_feed,
-                                           control_feed=self.control_feed,
-                                           history=self.history,
+            (r"/price", PriceHandler, dict(pairs=pairs,
+                                           configs=configs,
                                            cache=self.cache)),
             (r"/deal", DealHandler, dict(cache=self.cache,
                                          schema=deal_schema())),
         ])
         application.listen(port=self.arguments.http_port,address=self.arguments.http_address)
         tornado.ioloop.IOLoop.current().start()
 
+    #
+    # Multiple markets configuration sample
+    #
+    # {
+    #     "markets": [
+    #         {
+    #             "pair": "ETH/DAI",
+    #             "bands": "~/imtoken-ethdai-bands.json",
+    #             "price-feed": "eth_dai-pair-midpoint",
+    #             "price-feed-expiry": 20
+    #         },
+    #         {
+    #             "pair": "MKR/DAI",
+    #             "bands": "~/imtoken-mkrdai-bands.json",
+    #             "price-feed": "ws://admin:admin@localhost:9595/api/feeds/MKR_DAI_PRICE/socket",
+    #             "price-feed-expiry": 20
+    #         }
+    #     ]
+    # }
+    @staticmethod
+    def parse_configs(data: dict) -> (list, dict):
```

Comment:
Does this need to be @staticmethod? Also, where is this called outside the class, I can't find it.

If it isn't lets rename it to `_parse_configs()`

Reply:
It is fine to be static as we just want to parse / create config and not change any state, good point on renaming function, will change that